src/Routes.js

```diff
@@ -99,6 +100,7 @@ const Routes = (props) => (
       props.env === 'dev' ? <AuthenticatedRoute path="/new_member_console" component={NewMemberConsole} props={props} rolesAllowed={[USER_ROLES.member]}/>
         : ''
     }
+    <AuthenticatedRoute path="/dar_vote_review/:collectionId" component={DarVoteReview} props={props} rolesAllowed={[USER_ROLES.chairperson, USER_ROLES.member]}/>
```

Comment:
This comment is a continuation of the one left on `DarVoteReview`. Instead of creating a new file that just adds `readOnly` to props for `DarCollectionReview`, it'll be easier to add `readOnly` to props here and set `component={DarCollectionReview}`